crates/web-sys/tests/all/response.rs

```diff
@@ -0,0 +1,37 @@
+use super::project;
+
+#[test]
+fn response() {
+    project()
+        .add_local_dependency("web-sys", env!("CARGO_MANIFEST_DIR"))
+        .headless(true)
```

Comment:
We are going to end up repeating these three lines in every single test in this crate -- would you be interested in writing a follow up PR that defines a helper function to preconfigure these bits in `crates/web-sys/tests/all/main.rs` ?

Reply:
Certainly! :)